thirdeye/thirdeye-frontend/app/styles/pods/rootcause.scss

```diff
@@ -42,3 +42,9 @@
     margin-left: 8px;
   }
 }
+
+.rootcause-alert {
+  margin-left: $te-spacing-lg;
```

Comment:
are you applying the property separately (instead of the shorthand) because of bootstrap's stronger selector?